avm/res/databricks/workspace/main.bicep

```
@@ -156,14 +170,26 @@ resource cMKManagedDiskKeyVault 'Microsoft.KeyVault/vaults@2023-07-01' existing
   }
 }
 
-resource workspace 'Microsoft.Databricks/workspaces@2023-02-01' = {
+// This var is a workaround for the fact both properties can not be null or empty, they either exist or not in the properties object.
+var defaulStorageFirewallProperties = !empty(privateStorageAccount)
```

```suggestion
var defaultStorageFirewallProperties = !empty(privateStorageAccount)
```
References must be fixed too